doc/source/docs/howto_build_docs.rst

```diff
@@ -48,13 +48,22 @@ with::
 
 Now you are ready to generate the docs, so write::
 
+    cd doc
     make html
 
 in the ``doc/`` directory. If all goes well, this will generate a
 ``build/html`` subdirectory containing the built documentation. If you get
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
+Note that if you have built numpy into a virtual enviornment and getting
+an error that says ``numpy not found, cannot build documentation without successful "import numpy"``
```

Comment:
Please keep the lines < 80 characters.

Reply:
Maybe use ``numpy not found, cannot build documentation without . . .``

Reply:
Thanks for the assist @charris, I'll make these changes, although could you please clarify what you mean by

> Maybe use ` ``numpy not found, cannot build documentation without . . .`` `

should I replace ` ``numpy not found, cannot build documentation without successful "import numpy"`` ` with exactly what you said?

Reply:
That is the suggestion, the ellipsis indicates that text has been omitted. I looked for ways to break inline literals and could not find any and even putting it on a new line was a tiny bit over, although probably acceptable. The other option was to change the error message itself, but that seemed overkill.